Xamarin.PropertyEditing.Mac/Controls/SizeEditorControl.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections;
+using System.Diagnostics;
+using System.Drawing;
+using AppKit;
+using CoreGraphics;
+using Foundation;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class SizeEditorControl : BasePointEditorControl<CGSize>
+	{
+		public SizeEditorControl ()
+		{
+			XLabel.Frame = new CGRect (0, 0, 40, 24);
+			XLabel.StringValue = "Width:"; // TODO Localise
+
+			XEditor.Frame = new CGRect (45, 0, 50, 20);
+			XEditor.Activated += (sender, e) => {
```

Comment:
We should consider the hooks in the base class that calls a virtual method to update the value as we should _also_ be updating value on blur.